regtests/bin/matrix_cmake_ncep

```diff
@@ -43,11 +43,8 @@ fi
 #convert main_dir to absolute path
 main_dir="`cd $main_dir 1>/dev/null 2>&1 && pwd`"
 
-# Compiler option. Choose appropriate compiler and set cmplOption to
-# y if using for the first time or using a different compiler
+#download 
```

Comment:
do we need this comment?

Reply:
No we do not need this anymore.  Good catch!

Reply:
This has been updated and i cleaned up some other formatting while i was double checking things.